integration-tests/application-server-integration-tests/src/test/java/co/elastic/apm/servlet/AbstractServletContainerIntegrationTest.java

```diff
@@ -154,6 +154,7 @@ protected AbstractServletContainerIntegrationTest(GenericContainer<?> servletCon
             .withEnv("ELASTIC_APM_TRACE_METHODS", "public @@javax.enterprise.context.NormalScope co.elastic.*")
             .withEnv("ELASTIC_APM_DISABLED_INSTRUMENTATIONS", "") // enable all instrumentations for integration tests
             .withEnv("ELASTIC_APM_PROFILING_SPANS_ENABLED", "true")
+            .withEnv("ELASTIC_APM_APPLICATION_PACKAGES", "co.elastic") // allows to use API annotations, we have to use a broad package due to multiple apps
```

Comment:
just out of curiosity: why would `co.elastic.webapp` not work?

Reply:
Other applications were in other sub-packages, for example `co.elastic.apm.spring.boot` or `co.elastic.apm.soap`, and setting it to the `webapp` sub-package breaks the `soap` test application as it makes soap transactions ignored (`co.elastic.apm.soap.HelloWorldServiceImpl` is annotated with `@javax.jws.WebService`).

There is a small difference here when the `application_packages` is not set, the `@javax.jws.WebService` is properly instrumented, whereas methods annotated with `@co.elastic.apm.api.CaptureSpan` are not.

I think it might be worth making it consistent and make `@CaptureSpan` and other annotations instrumentation work by default (even if it makes things slower), but that should be fixed in another PR.